AU/Public/Update-AUPackages.ps1

```diff
@@ -130,7 +130,7 @@ function Update-AUPackages {
                     }
                 }
 
-
+                $jobseconds = ($job.PSEndTime.TimeOfDay - $job.PSBeginTime.TimeOfDay).TotalSeconds
```

Comment:
I wouldn't have use `TimeOfDay` here (just in case we are updating at around midnight):
`$jobseconds = ($job.PSEndTime - $job.PSBeginTime).TotalSeconds`

Reply:
(I know it's already merged though).

Reply:
ugh - i didn't think of that scenario with `timeofday` I was running w/ this for a while (obviously never at midnight).